ods_ci/tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -480,18 +494,15 @@ Remove Items From Favorites
     Close Browser
 
 RHODS Dahsboard Pod Should Contain OauthProxy Container
-    ${list_of_pods} =    Search Pod    namespace=${APPLICATIONS_NAMESPACE}    pod_start_with=rhods-dashboard
+    ${list_of_pods} =    Search Pod    namespace=${APPLICATIONS_NAMESPACE}    pod_regex=${DASHBOARD_APP_NAME}
     FOR    ${pod_name}    IN   @{list_of_pods}
         ${container_name} =    Get Containers    pod_name=${pod_name}    namespace=${APPLICATIONS_NAMESPACE}
         List Should Contain Value    ${container_name}    oauth-proxy
     END
 
 Verify Jupyter Card CSS Style
-    [Documentation]    Compare the some CSS properties of the Explore page
-    ...    with the expected ones. The expected values change based
-    ...    on the RHODS version
-    CSS Property Value Should Be    locator=//pre
-    ...    property=background-color    exp_value=rgba(240, 240, 240, 1)
```

Comment:
iirc, all the text inside a //pre is a code? I agree the color might change, but this could capture formatting issues of special text

Reply:
The //pre is not available anymore...
I did find other elements with background-color but I think that verifying a specific color, not based on an actual reference of expected color, would make the test fragile.

Reply:
I re-added the verification of the css color of the explorer drawer panel by test id

Reply:
@bdattoma I changed the logic to select the Jupyter card specifically (instead of the default first card) and then verify its code background-color. (not all cards have code, as the default selected card).